src/IssueViz.UnitTests/Editor/LocationTagging/IssueLocationStoreAggregatorTests.cs

```diff
@@ -36,13 +36,24 @@ namespace SonarLint.VisualStudio.IssueVisualization.UnitTests.Editor.LocationTag
     public class IssueLocationStoreAggregatorTests
     {
         [TestMethod]
-        public void MefCtor_CheckExports()
+        public void MefCtor_CheckIsExported()
+        {
+            // IIssueLocationStores are ImportMany -> optional
+            MefTestHelpers.CheckTypeCanBeImported<IssueLocationStoreAggregator, IIssueLocationStoreAggregator>();
+        }
+
+        [TestMethod]
+        public void MefCtor_CheckIsExported_LocationStoresAreImported()
```

Comment:
I modified the existing test to check that the expected `ImportMany` types are actually imported.